molgenis-data/src/main/java/org/molgenis/data/meta/model/Attribute.java

```diff
@@ -199,6 +199,10 @@ public boolean isLabelAttribute()
 	public Attribute setLabelAttribute(Boolean isLabelAttr)
 	{
 		set(IS_LABEL_ATTRIBUTE, isLabelAttr);
+		if (isLabelAttr != null && isLabelAttr)
```

Comment:
We ususally don't put the logic in the entity class. Will be confusing that this setter has side effects:
`addAttribute("blah").setNillable(true).setLabelAttribute(true)`